lunatrace/bsl/backend/src/topic-ingester/schemas/generated-schema-1-validator.ts

```diff
@@ -231,12 +197,20 @@ const typeMap: any = {
         { json: "purl", js: "purl", typ: u(undefined, "") },
         { json: "versionConstraint", js: "versionConstraint", typ: "" },
         { json: "name", js: "name", typ: u(undefined, "") },
-        { json: "language", js: "language", typ: u(undefined, "") },
     ], false),
     "Cwe": o([
         { json: "number", js: "number", typ: 0 },
         { json: "name", js: "name", typ: "" },
     ], false),
+    "Package": o([
+        { json: "type", js: "type", typ: "" },
+        { json: "purl", js: "purl", typ: "" },
+        { json: "language", js: "language", typ: "" },
+        { json: "name", js: "name", typ: "" },
+        { json: "versionConstraint", js: "versionConstraint", typ: "" },
+        { json: "fixed", js: "fixed", typ: true },
```

Comment:
Will `typ` be smart enough to know that `true` means that this field is a boolean or will it enforce that this field always be `true`?

Reply:
thats how it works, it just does `typeof predicate === typeof whateverThing`

Reply:
Thats why all the strings are of type ""